pattern-1/bosh-release/config/blobs.yml

```diff
@@ -0,0 +1,20 @@
+jdbcdrivers/mssql-jdbc-7.0.0.jre8.jar:
```

Comment:
Shall we keep this file empty as these SHA values change every time the BOSH release is created?